### ISSUE TITLE
Make line of sight independent from terrain level of details

### ISSUE THREAD

**Issue by ComBatVision** [WorldWindEarth/WorldWindKotlin#74]
Now line of sight is almost useless, because it depends on tessellator level of details.
When user looks on globe in coutry scale terrain is almost flat.
We need to calculate line of sight in the most detailed terrain level and make a texture from it.

**Comment by gokboeruealp** [comment]
Is there any improvement about this issue?

**Comment by ComBatVision** [comment]
No. It requires to develop completely another tool which will be based not on shadows, but on textures like heatmap.

**Comment by EMaksymenko** [comment]
Done. ViewshedSightline
